lib/ramble/ramble/software_environments.py

```diff
@@ -413,7 +413,8 @@ class SoftwareEnvironments(object):
     """Class representing a group of software environments"""
 
     _deprecated_sections = [namespace.variables, namespace.zips,
-                            namespace.matrix, namespace.matrices, namespace.exclude]
+                            namespace.matrix, namespace.matrices,
+                            namespace.exclude, 'concretized']
```

Comment:
I had asked Doug about this and he seemed to think it would be ok since it was in a 'deprecated' section and we'll remove it fairly soon.

Reply:
Yeah, I figured since it's going to be deleted soon, it wasn't necessarily worth adding into a variable. The other variables were already defined, and are still useful after `concretized` is removed.